src/asset-swapper/utils/market_operation_utils/types.ts

```diff
@@ -212,15 +211,13 @@ export interface KyberDmmFillData extends UniswapV2FillData {
 /**
  * Determines whether FillData is UniswapV3FillData or FinalUniswapV3FillData
  */
-export function isFinalUniswapV3FillData(
-    data: UniswapV3FillData | FinalUniswapV3FillData,
-): data is FinalUniswapV3FillData {
-    return !!(data as FinalUniswapV3FillData).uniswapPath;
+export function isFinalPathFillData(data: MultiPathFillData | FinalPathFillData): data is FinalPathFillData {
+    return !!(data as FinalPathFillData).path;
 }
 
-export interface FinalUniswapV3FillData extends Omit<UniswapV3FillData, 'pathAmounts'> {
-    // The uniswap-encoded path that can fll the maximum input amount.
-    uniswapPath: string;
+export interface FinalPathFillData extends Omit<MultiPathFillData, 'pathAmounts'> {
```

Comment:
nit/optional: I think the name is bit too generic 🤔 

Given it's name I think it's hard to know that it's for Uniswap V3/Tick-based AMMs (not sure what would be a good name though without making it too long 😔 `FinalUniswapV3LikePathFillData` is kind of long).



Another option could be creating an alias. 

```
type KyberElasticMultiPathFillData = UniswapV3MultiPathFillData
```

Reply:
`TickAMMMultiPathFillData` ?

Reply:
I'll go with `TickDEXMultiPathFillData` and `FinalTickDEXMultiPathFillData` unless any objections. I'll go with the slightly too long name if it's clearer.